event_docs.go

```diff
@@ -0,0 +1,73 @@
+package gosparkpost
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+var eventDocumentationFormat = "/api/v%d/webhooks/events/documentation"
+
+type EventGroup struct {
+	Name        string
+	Events      map[string]EventMeta
```

Comment:
Would it be more consistent to include `json:"events"` for `Events      map[string]EventMeta`?